src/components/modules/ui.ts

```diff
@@ -477,6 +477,12 @@ export default class UI extends Module<UINodes> {
    * @param {KeyboardEvent} event - keyboard event
    */
   private documentKeydown(event: KeyboardEvent): void {
```

Comment:
this method is designed to handle keydowns on document. I'm afraid that this change break something like inserting new block on Enter when no block is focused, etc. 

The possible problem place is the `BlockEvents@slachPressed`

```
this.Editor.Caret.insertContentAtCaretPosition('/');
```

So it is expected to see a fix there.

Reply:
@neSpecc I have moved the fix to `BlockEvents`

Reply:
@neSpecc Have you had some time to check this again?

Reply:
Hello @neSpecc, have you had a chance to check this again?